Nexpo/Controllers/CompaniesController.cs

```diff
@@ -165,6 +165,32 @@ public async Task<ActionResult> PutMe(UpdateCompanySelfDto dto)
             }
             await _companyRepo.Update(company);
 
+            return Ok(company);
+        }
+
+         /// <summary>
+        /// Add new company
+        /// </summary>
+        [HttpPut]
+        [Route("add")]
+        [Authorize(Roles = nameof(Role.Administrator))]
+        [ProducesResponseType(typeof(Company), StatusCodes.Status200OK)]
+        public async Task<ActionResult> AddNewCompany(AddCompanyDto dto)
+        {
+            
+            var company = new Company {
```

Comment:
In the company table exists columns/attributes connected to the responsible host of the company at the event. Are these details not provided at the moment and is it something that will be provided at a later date by updating the company information?

Reply:
Good point, however i do not think the information about the hosts will be added to jexpo, therefore i decided to leave it out. Maybe I should add it anyways?

Reply:
Hm... Since you have declared all the attributes in the AddCompanyDto it may cause future problems as the provided host-information currently always gets ignored by the controller.  So for consistency I would either delete the attributes from the DTO or add them in the controller. :)

Reply:
I 100% agree